emva1288/process/plotting.py

```diff
@@ -422,8 +422,8 @@ def plot(self, test):
                 label='Data',
                 gid='%d:data' % test.id)
 
-        ax.axhline(test.DSNU1288_DN(),
-                   label='$DSNU_{1288.DN}$',
+        ax.axhline(test.DSNU1288,
+                   label='$DSNU_{1288}$',
```

Comment:
The y-axis units are DN, but `test.DSNU1288` units are $e^-$, so there is a mismatch.

Reply:
Good catch, DN unit of DSNU has been deprecated, so I need to update all units for DSNU to e-.

Reply:
How will you convert the spectrogram data and `np.sqrt(test.sigma_2_y_stack_dark)` to $e^-$?

Reply:
That's a good question, previously there is a function that calculates the DSNU_DN, but in the new version, it's no longer needed, so I removed the DSNU_DN and use DSNU directly. 
" The DSNU1288 is expressed in units e−; by multiplying with the overall system gain K it can also be given in units DN."
So I believe it is being transferred by the system gain.

Reply:
The system gain has a unit in "DN/e-", so this is the key to conversion.

Reply:
Unresolving until all the units actually match.

Reply:
sure